apps/backend/src/app/api/v1/auth/passkey/sign-in/verification-code-handler.tsx

```diff
@@ -80,29 +82,35 @@ export const passkeySignInVerificationCodeHandler = createVerificationCodeHandle
 
     if (!isLocalhost) {
       if (!project.config.domains.map(e => e.domain).includes(parsedOrigin.origin)) {
-        throw new KnownErrors.PasskeyAuthenticationFailed();
+        throw new KnownErrors.PasskeyAuthenticationFailed("Passkey authentication failed because the origin is not allowed");
       } else {
         expectedRPID = parsedOrigin.hostname;
         expectedOrigin = origin;
       }
     }
 
-    const authVerify = await verifyAuthenticationResponse({
-      response: authenticationResponse,
-      expectedChallenge: challenge,
-      expectedOrigin,
-      expectedRPID,
-      credential: {
-        id: passkey.userHandle,
-        publicKey: new Uint8Array(Buffer.from(passkey.publicKey, 'base64')),
-        counter: passkey.counter,
-      },
-      requireUserVerification: false,
-    });
+    let authVerify;
+    try {
+      authVerify = await verifyAuthenticationResponse({
+        response: authentication_response,
+        expectedChallenge: challenge,
+        expectedOrigin,
+        expectedRPID,
+        credential: {
+          id: passkey.userHandle,
+          publicKey: new Uint8Array(Buffer.from(passkey.publicKey, 'base64')),
+          counter: passkey.counter,
+        },
+        requireUserVerification: false,
+      });
+    } catch (error) {
+      captureError("Failed to verify passkey authentication response", error);
+      throw new KnownErrors.PasskeyAuthenticationFailed("Failed to verify passkey authentication response");
```

Comment:
Originally I did this because I did not want to return errors with potentially sensitive errors, but looking at it again, none of the errors that the library throws are sensitive, like expectedChallenge or the list of origins that are accepted. I guess we can remove the sentry logging and just return the error directly

Reply:
We should get rid of the catch-all, then, and catch the specific errors that the library can throw.

Btw, the first argument of `captureError` should be a machine-readable ID of the error location, eg. `"passkey-authentication-response-verification"`